src/libraries/System.Private.CoreLib/src/System/IO/Strategies/FileStreamHelpers.cs

```diff
@@ -11,17 +11,7 @@ internal static partial class FileStreamHelpers
         internal static bool UseNet5CompatStrategy { get; } = GetNet5CompatFileStreamSetting();
 
         private static bool GetNet5CompatFileStreamSetting()
-        {
-            if (AppContext.TryGetSwitch("System.IO.UseNet5CompatFileStream", out bool fileConfig))
-            {
-                return fileConfig;
-            }
-
-            string? envVar = Environment.GetEnvironmentVariable("DOTNET_SYSTEM_IO_USENET5COMPATFILESTREAM");
-            return envVar is null
-                ? true // Net5Compat is currently enabled by default;
-                : bool.IsTrueStringIgnoreCase(envVar) || envVar.Equals("1");
-        }
+            => AppContextConfigHelper.GetBooleanConfig("System.IO.UseNet5CompatFileStream", "DOTNET_SYSTEM_IO_USENET5COMPATFILESTREAM");
```

Comment:
Is this helper method still needed?
```C#
internal static bool UseNet5CompatStrategy { get; } =
    AppContextConfigHelper.GetBooleanConfig("System.IO.UseNet5CompatFileStream", "DOTNET_SYSTEM_IO_USENET5COMPATFILESTREAM");
```

Reply:
Yes as it's reused by globalization as well:

https://github.com/dotnet/runtime/blob/6ef4b2e7aba70c514d85c2b43eac1616216bea55/src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs#L11-L18

Reply:
By helper, I mean GetNet5CompatFileStreamSetting().

Reply:
> By helper, I mean GetNet5CompatFileStreamSetting()

It's not, I've just pushed a commit that removes it.